src/test/groovy/sprouts/Properties_List_Spec.groovy

```diff
@@ -712,4 +712,107 @@ class Properties_List_Spec extends Specification
             vars.indexOf(null) == 1
     }
 
+    def 'You can easily create an empty "Vals" list.'() {
+        reportInfo """
+            A common use case for empty lists and nullable empty lists is inside the change delegate.
+            A delegate holds information about the change, such as the `newValues` and `oldValues`.
+            These can also be empty. For example, when a property is removed from the list, the newVals list is empty. 
+        """
+        given : 'A "Vals" and a nullable "Vals" instance with no properties.'
+            var vals = Vals.of(String.class)
+            var valsNullable = Vals.ofNullable(String.class)
+        expect : 'The `size` method returns 0 and the `isEmpty` method returns true.'
+            vals.size() == 0
+            valsNullable.size() == 0
+            vals.isEmpty()
+            valsNullable.isEmpty()
```

Comment:
It may also be good to check if their `type()` methods return `String`
since that is the only thing that distinguishes them from other empty property lists.